legal/satzung.md

```diff
@@ -0,0 +1,122 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
```

Comment:
If we decide the statute as suggested here, it won't be necessary to go there regularly: Most things can be done by mail and it would be enough that one member of the board would have her/his signature verified.

Reply:
Optionen:
 * Berlin
 * Flensburg
 * Bremen
 * Oldenburg
 
Berlins Verwaltungs ist eher langsam. Dafür kommt man immer mal nach Berlin.